src/Akka.Streams.Kafka/Helpers/PartitionEventHandler.cs

```diff
@@ -69,54 +76,69 @@ public void OnAssign(IImmutableSet<TopicPartition> assignedTopicPartitions, IRes
             public void OnStop(IImmutableSet<TopicPartition> topicPartitions, IRestrictedConsumer consumer)
             {
             }
+
+            public override string ToString() => "EmptyHandler";
         }
         
         /// <summary>
         /// Handler allowing to pass custom stage callbacks.
         /// </summary>
-        internal class AsyncCallbacks : IPartitionEventHandler
+        internal sealed class AsyncCallbacks : IPartitionEventHandler
         {
+            private readonly IAutoSubscription _subscription;
+            private readonly IActorRef _sourceActorRef;
+            
             private readonly Action<IImmutableSet<TopicPartition>> _partitionAssignedCallback;
             private readonly Action<IImmutableSet<TopicPartitionOffset>> _partitionRevokedCallback;
-            private readonly Action<IImmutableSet<TopicPartitionOffset>> _partitionLostCallback;
```

Comment:
We do not need a separate "partitions lost" handler - just calling the "partitions revoked" handler when a partitions lost event is raised is sufficient.